From 1bae65bed996a5dfb0414a52dd0e24f9d2cf984b Mon Sep 17 00:00:00 2001
From: "Hebbar, Gururaja" <gururaja.hebbar@ti.com>
Date: Tue, 31 Jan 2012 08:55:52 +0530
Subject: [PATCH 001/186] arm:omap:am33xx - add missing i2c pin mus details

Add all references to i2c pins in mux file. These pins may be used in
different board configurations.

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
---
 arch/arm/mach-omap2/mux33xx.c |   24 ++++++++++++------------
 1 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/arch/arm/mach-omap2/mux33xx.c b/arch/arm/mach-omap2/mux33xx.c
index 81a657e..8df8fa9 100644
--- a/arch/arm/mach-omap2/mux33xx.c
+++ b/arch/arm/mach-omap2/mux33xx.c
@@ -231,10 +231,10 @@ static struct omap_mux __initdata am33xx_muxmodes[] = {
 		"mii1_col", "rmii2_refclk", "spi1_sclk", NULL,
 		"mcasp1_axr2", "mmc2_dat3", "mcasp0_axr2", "gpio3_0"),
 	_AM33XX_MUXENTRY(MII1_CRS, 0,
-		"mii1_crs", "rmii1_crs_dv", "spi1_d0", NULL,
+		"mii1_crs", "rmii1_crs_dv", "spi1_d0", "i2c1_sda",
 		"mcasp1_aclkx", NULL, NULL, "gpio3_1"),
 	_AM33XX_MUXENTRY(MII1_RXERR, 0,
-		"mii1_rxerr", "rmii1_rxerr", "spi1_d1", NULL,
+		"mii1_rxerr", "rmii1_rxerr", "spi1_d1", "i2c1_scl",
 		"mcasp1_fsx", NULL, NULL, "gpio3_2"),
 	_AM33XX_MUXENTRY(MII1_TXEN, 0,
 		"mii1_txen", "rmii1_txen", "rgmii1_tctl", NULL,
@@ -282,10 +282,10 @@ static struct omap_mux __initdata am33xx_muxmodes[] = {
 		"mdio_clk", NULL, NULL, NULL,
 		"mmc0_sdwp", "mmc1_clk", "mmc2_clk", "gpio0_1"),
 	_AM33XX_MUXENTRY(SPI0_SCLK, 0,
-		"spi0_sclk", "uart2_rxd", NULL, NULL,
+		"spi0_sclk", "uart2_rxd", "i2c2_sda", NULL,
 		NULL, NULL, NULL, "gpio0_2"),
 	_AM33XX_MUXENTRY(SPI0_D0, 0,
-		"spi0_d0", "uart2_txd", NULL, NULL,
+		"spi0_d0", "uart2_txd", "i2c2_scl", NULL,
 		NULL, NULL, NULL, "gpio0_3"),
 	_AM33XX_MUXENTRY(SPI0_D1, 0,
 		"spi0_d1", "mmc1_sdwp", "i2c1_sda", NULL,
@@ -300,28 +300,28 @@ static struct omap_mux __initdata am33xx_muxmodes[] = {
 		"ecap0_in_pwm0_out", "uart3_txd", "spi1_cs1", NULL,
 		"spi1_sclk", "mmc0_sdwp", NULL, "gpio0_7"),
 	_AM33XX_MUXENTRY(UART0_CTSN, 0,
-		"uart0_ctsn", NULL, "d_can1_tx", NULL,
+		"uart0_ctsn", NULL, "d_can1_tx", "i2c1_sda",
 		"spi1_d0", NULL, NULL, "gpio1_8"),
 	_AM33XX_MUXENTRY(UART0_RTSN, 0,
-		"uart0_rtsn", NULL, "d_can1_rx", NULL,
+		"uart0_rtsn", NULL, "d_can1_rx", "i2c1_scl",
 		"spi1_d1", "spi1_cs0", NULL, "gpio1_9"),
 	_AM33XX_MUXENTRY(UART0_RXD, 0,
-		"uart0_rxd", "spi1_cs0", "d_can0_tx", NULL,
+		"uart0_rxd", "spi1_cs0", "d_can0_tx", "i2c2_sda",
 		NULL, NULL, NULL, "gpio1_10"),
 	_AM33XX_MUXENTRY(UART0_TXD, 0,
-		"uart0_txd", "spi1_cs1", "d_can0_rx", NULL,
+		"uart0_txd", "spi1_cs1", "d_can0_rx", "i2c2_scl",
 		NULL, NULL, NULL, "gpio1_11"),
 	_AM33XX_MUXENTRY(UART1_CTSN, 0,
-		"uart1_ctsn", NULL, NULL, NULL,
+		"uart1_ctsn", NULL, NULL, "i2c2_sda",
 		"spi1_cs0", NULL, NULL, "gpio0_12"),
 	_AM33XX_MUXENTRY(UART1_RTSN, 0,
-		"uart1_rtsn", NULL, NULL, NULL,
+		"uart1_rtsn", NULL, NULL, "i2c2_scl",
 		"spi1_cs1", NULL, NULL, "gpio0_13"),
 	_AM33XX_MUXENTRY(UART1_RXD, 0,
-		"uart1_rxd", "mmc1_sdwp", NULL, NULL,
+		"uart1_rxd", "mmc1_sdwp", NULL, "i2c1_sda",
 		NULL, "pr1_uart0_rxd_mux1", NULL, "gpio0_14"),
 	_AM33XX_MUXENTRY(UART1_TXD, 0,
-		"uart1_txd", "mmc2_sdwp", NULL, NULL,
+		"uart1_txd", "mmc2_sdwp", NULL, "i2c1_scl",
 		NULL, "pr1_uart0_txd_mux1", NULL, "gpio0_15"),
 	_AM33XX_MUXENTRY(I2C0_SDA, 0,
 		"i2c0_sda", NULL, NULL, NULL,
-- 
1.7.9.1

